modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ScopeContainer.tsx

```diff
@@ -134,6 +134,7 @@ export function ScopeContainer({
 					'no-options-were-found'
 				)}
 				error={errors.titleObjectFieldId}
+				id="scope-container"
```

Comment:
Hi @Artartur. Please, rename the id to `objectDetailsScopeContainer`.

Reply:
Done!